### PR TITLE
first update on docstrings

### PR DESCRIPTION
**Pull request by keunwoochoi**
@rabitt Sorry it's really late. Some unfilled args are listed on #138. A separate topic is at #142. Let me know if I'm missing something obvious.

### CODE REVIEW

File: mirdata/ikala.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ class Track(object):
         track_id (str): track id
         audio_path (str): track audio path
         song_id (str): song id of the track
-        section (str): section (todo)
+        section (str): section
```
Comment (rabitt) [inline mirdata/ikala.py]: it's one of 'verse' or 'chorus'

File: mirdata/medleydb_melody.py [hunk 2]
```diff
@@ -69,7 +69,7 @@ class Track(object):
         title (str): title of the track
         genre (str): genre of the track
         is_excerpt (bool):
```
Comment (rabitt) [inline mirdata/medleydb_melody.py]: True if the track is an excerpt

File: mirdata/medleydb_melody.py [hunk 3]
```diff
@@ -69,7 +69,7 @@ class Track(object):
...
-        is_instrumental (bool)
+        is_instrumental (bool):
```
Comment (rabitt) [inline mirdata/medleydb_melody.py]: True if the track does not contain vocals

File: mirdata/medleydb_melody.py [hunk 4]
```diff
@@ -69,7 +69,7 @@ class Track(object):
...
         n_sources (int):
```
Comment (rabitt) [inline mirdata/medleydb_melody.py]: Number of instruments in the track

File: mirdata/medleydb_melody.py [hunk 5]
```diff
@@ -69,7 +69,7 @@ class Track(object):
...
         melody1 (F0Data):
```
Comment (rabitt) [inline mirdata/medleydb_melody.py]: Melody 1: the pitch of the single most predominant source (often the voice)
Melody 2: the pitch of the predominant source for each point in time
Melody 3: the pitch of any melodic source. Allows for more than one f0 value at a time.

File: mirdata/orchset.py [hunk 6]
```diff
@@ -111,24 +111,24 @@ class Track(object):
...
     Attributes:
-        track_id (str): track id
-        audio_path_mono (str): mono audio path of the track
-        audio_path_stereo (str): stereo audio path of the track
-        composer (str): composer
+        track_id (str): Track id
+        audio_path_mono (str): Mono audio path of the track
+        audio_path_stereo (str): Stereo audio path of the track
+        composer (str): Composer of the track
         work (str):
```
Comment (rabitt) [inline mirdata/orchset.py]: The musical work

File: mirdata/orchset.py [hunk 7]
```diff
@@ -111,24 +111,24 @@ class Track(object):
...
         predominant_melodic_instruments (str):
```
Comment (rabitt) [inline mirdata/orchset.py]: List of instruments which play the melody

File: mirdata/orchset.py [hunk 8]
```diff
@@ -111,24 +111,24 @@ class Track(object):
...
-        alternating_melody
-        contains_winds (bool?)
-        contains_strings
-        contains_brass
-        only_strings
-        only_winds
-        only_brass
+        alternating_melody:
```
Comment (rabitt) [inline mirdata/orchset.py]: bool - true if the melody alternates between instruments

File: mirdata/rwc_jazz.py [hunk 9]
```diff
@@ -80,6 +85,26 @@ def _load_metadata(data_home):
...
+    Attributes:
+        track_id (str): Track id
+        audio_path (str): Audio path of this Track
+        piece_number (str): Piece number of this Track, [1-50]
+        suffix (str): M01-M04
+        track_number: CD track number of this Track
+        title (str): Title of The track.
+        artist (str): Artist name with the vocal's gender
+            E.g., 'Makoto Nakamura'
+        duration_sec: Duration of the track in seconds
```
Comment (rabitt) [inline mirdata/rwc_jazz.py]: (float)

File: mirdata/rwc_jazz.py [hunk 10]
```diff
@@ -80,6 +85,26 @@ def _load_metadata(data_home):
...
+        variation:
```
Comment (rabitt) [inline mirdata/rwc_jazz.py]: @magdalenafuentes ?

File: mirdata/rwc_jazz.py [hunk 11]
```diff
@@ -80,6 +85,26 @@ def _load_metadata(data_home):
...
+        instruments: list of used instruments.
```
Comment (rabitt) [inline mirdata/rwc_jazz.py]: instruments (list)

File: mirdata/rwc_popular.py [hunk 12]
```diff
@@ -95,6 +100,28 @@ def _load_metadata(data_home):
...
+    Attributes:
+        track_id (str): Track id
+        audio_path (str): Audio path of this Track
+        piece_number (str): Piece number of this Track, [1-50]
+        suffix (str): M01-M04
+        track_number: CD track number of this Track
+        title (str): Title of The track.
+        artist (str): Artist name with the vocal's gender
+            E.g., 'Makoto Nakamura'
+        duration_sec: Duration of the track in seconds
```
Comment (rabitt) [inline mirdata/rwc_popular.py]: (float)

File: mirdata/rwc_popular.py [hunk 13]
```diff
@@ -95,6 +100,28 @@ def _load_metadata(data_home):
...
+        tempo: Tempo of the track in BPM
```
Comment (rabitt) [inline mirdata/rwc_popular.py]: (float)

File: mirdata/rwc_popular.py [hunk 14]
```diff
@@ -95,6 +100,28 @@ def _load_metadata(data_home):
...
+        instruments: List of used instruments
+        drum_information: If the drum is 'Drum sequences', 'Live drums',
```
Comment (rabitt) [inline mirdata/rwc_popular.py]: (str)

File: mirdata/rwc_popular.py [hunk 15]
```diff
@@ -95,6 +100,28 @@ def _load_metadata(data_home):
...
+        audio_path (str): Audio path of this Track
+        piece_number (str): Piece number of this Track, [1-50]
+        suffix (str): M01-M04
+        track_number: CD track number of this Track
+        title (str): Title of The track.
+        artist (str): Artist name with the vocal's gender
+            E.g., 'Makoto Nakamura'
+        duration_sec: Duration of the track in seconds
+        tempo: Tempo of the track in BPM
+        instruments: List of used instruments
```
Comment (rabitt) [inline mirdata/rwc_popular.py]: (list)

**Comment by keunwoochoi** [comment]
done!

**Pull request merged**